test/distributed/org/apache/cassandra/service/accord/AccordJournalBurnTest.java

```diff
@@ -115,6 +115,11 @@ public static void setUp() throws Throwable
         FieldUtil.setInstanceUnsafe(TopologySerializers.class,
                                     new TopologySerializers.ShardSerializer(BurnTestKeySerializers.range),
                                     "shard");
+        // compact topology inlines all serialziation and uses TokenRange directly, so it has to be fully stubbed out
+        // for this class.
+        FieldUtil.setInstanceUnsafe(TopologySerializers.class,
```

Comment:
trying to make this flexible seemed a bit too much as i need to do a larger refactor over burn test and try to remove all our serialize overrides.